sdk/core/azure-core/tests/test_rest_http_request.py

```diff
@@ -550,16 +551,16 @@ def test_multipart_incorrect_tuple_entry(filebytes):
 def test_multipart_tuple_input_single(filebytes):
     request = HttpRequest("POST", url="http://example.org", files=[("file", filebytes)])
 
-    assert request.content == {"file": ("conftest.py", filebytes, "application/octet-stream")}
+    assert request.content == [("file", ("conftest.py", filebytes, "application/octet-stream"))]
 
 
 def test_multipart_tuple_input_multiple(filebytes):
     request = HttpRequest("POST", url="http://example.org", files=[("file", filebytes), ("file2", filebytes)])
 
-    assert request.content == {
-        "file": ("conftest.py", filebytes, "application/octet-stream"),
-        "file2": ("conftest.py", filebytes, "application/octet-stream"),
-    }
+    assert request.content == [
+        ("file", ("conftest.py", filebytes, "application/octet-stream")),
+        ("file2", ("conftest.py", filebytes, "application/octet-stream")),
+    ]
```

Comment:
will this change in format affect folks who were expecting a dict ?

Reply:
This change only applies if you pass in a list of tuples of files, which we previously would crash on, so old users, who would input `files={"file": filebytes}` wouldn't see any difference. Only users that wanted to input `files=[(file, filebytes)]`, and would get an iteration error (bc we just assumed dictionary), would get a list of tuples back. So no one is broken

Reply:
awesome, thank you :)